lib/Conversation/Comment.js

```diff
@@ -113,8 +113,7 @@ class Comment extends Component {
     const userCanEdit = createdBy === user.id && userCanComment;
     const className = cx('conversation__comment', {
       'is-visually-disabled': this.state.confirmRemoval,
-      'has-failed': this.props.hasFailed,
-      'has-author-type': author && author.type
+      'has-failed': this.props.hasFailed
```

Comment:
I have removed the `author-type` class from comments, the author type is now in brackets next to the users name!